mlflow/server/js/src/common/components/long-form/LongFormSection.tsx

```diff
@@ -10,22 +10,30 @@ export interface LongFormSectionProps {
   children: ReactNode;
   /** Whether to hide the bottom border divider (default: false) */
   hideDivider?: boolean;
+  /** Whether to hide the top padding (default: false, useful for first section in embedded contexts) */
+  hideTopPadding?: boolean;
```

Comment:
tip: a more flexible and idiomatic solution is to provide `className` prop, (and pass it to the overarching `<div>` within `LongFormSection`) which allows to override styles of any component using emotion's `css={...}` prop.